components/resources/brave_news_strings.grdp

```diff
@@ -7,7 +7,7 @@
     Follow your favorite sources, in a single feed. Just open a tab in Brave, scroll down, and… voila!
   </message>
   <message name="IDS_BRAVE_NEWS_INTRO_DESCRIPTION_TWO" desc="Description second paragraph for introduction to Brave News news feed section. Do not translate the title Brave News.">
-    Brave News is ad-supported with private, anonymized ads. <ph name="LINK_START">$1<ex>{</ex></ph>Learn more.<ph name="LINK_END">$2<ex>}</ex></ph>.
+    Brave News is ad-supported with private, anonymized ads. <ph name="LINK_START">$1<ex>{</ex></ph>Learn more.<ph name="LINK_END">$1<ex>}</ex></ph>.
```

Comment:
I am not sure why we have `{` and `}` as `<ex>`amples here. I think we can just get rid of them.

Reply:
Removed!